### ISSUE TITLE
NTR: Recently in area with inadequate sewage sanitation (Immune system assessed clinically)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8046]
New term request Recently in area with inadequate sewage sanitation (samant_110207033507):   UMLS: 0425226

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000540
- label: History of recent stay in area with inadequate sewage sanitation
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: 
- def: A history of having been in an area with inadequate sewage sanitation in the recent past.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8046
- synonyms:
